tests/custom_md_repo/testdata/model-settings.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
We don't need this one, right?

Reply:
Unless it changes anything over the root one, I would just let the tests reuse the other one.

Reply:
Done